apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -26,12 +84,65 @@ export default class TeacherDashboardHeader extends React.Component {
           isRtl={true}
           chevronSide="left"
         />
-        <div style={styles.headerContainer}>
-          <h1>{this.props.sectionName}</h1>
-          <SelectSectionDropdown />
+        <div style={styles.header}>
+          <div>
+            <h1>{this.selectedSection.name}</h1>
+            <div>
+              <span style={styles.sectionPrompt}>
+                {i18n.assignedToWithColon()}{' '}
+              </span>
+              {this.props.selectedSectionScript.name}
```

Comment:
You could combine these strings and pass the script name as a variable like this: https://github.com/code-dot-org/code-dot-org/blob/615055ffc00c85ef4ddc4f488a855972cfd91929/apps/i18n/common/en_us.json#L95.

Reply:
Will ask you more about this over slack